MeetupPlatformApi/Features/Meetups/RemoveMeetupContact/RemoveMeetupContactFeature.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace MeetupPlatformApi.Features.Meetups.RemoveMeetupContact;
+
+using AutoMapper;
+using MeetupPlatformApi.Persistence.Context;
+using MeetupPlatformApi.Seedwork.WebApi;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+
+[ApiSection(ApiSections.Meetups)]
+public class RemoveMeetupContactFeature : FeatureBase
```

Comment:
Plese, create class into contacts folder. You have relationship, therefore meetup won't have contact after delete.